src/tools/invert/inverttool.cpp

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-FileCopyrightText: 2017-2019 Alejandro Sirgo Rica & Contributors
+
+#include "inverttool.h"
+#include <QApplication>
+#include <QGraphicsBlurEffect>
+#include <QGraphicsPixmapItem>
+#include <QGraphicsScene>
+#include <QImage>
+#include <QPainter>
+#include <QPixmap>
+
+InvertTool::InvertTool(QObject* parent)
+  : AbstractTwoPointTool(parent)
+{}
+
+QIcon InvertTool::icon(const QColor& background, bool inEditor) const
+{
+    Q_UNUSED(inEditor)
+    return QIcon(iconPath(background) + "invert.svg");
+}
+
+QString InvertTool::name() const
+{
+    return tr("Invert");
+}
+
+ToolType InvertTool::type() const
+{
+    return ToolType::INVERT;
+}
+
+QString InvertTool::description() const
+{
+    return tr("Set Inverter as the paint tool");
+}
+
+CaptureTool* InvertTool::copy(QObject* parent)
+{
+    auto* tool = new InvertTool(parent);
+    copyParams(this, tool);
+    return tool;
+}
+
+void InvertTool::process(QPainter& painter, const QPixmap& pixmap)
+{
+    QPoint p0 = points().first;
+    QPoint p1 = points().second;
+    QRect selection = QRect(p0, p1).normalized();
+
+    // Invert selection
+    QPixmap inv = pixmap.copy(selection);
+    QImage img = inv.toImage();
+    img.invertPixels();
+
+    painter.drawImage(selection, img);
+}
+
+void InvertTool::drawSearchArea(QPainter& painter, const QPixmap& pixmap)
+{
+    Q_UNUSED(pixmap)
+    Q_UNUSED(painter)
```

Comment:
Reference the implementation in the Pixelate tool for this. Currently since you marked Q_UNUSED(painter), this new feature does not tie into the "movable" feature. IE I cannot move the selection once placed. 

If you duplicate the implementation that is in the pixelate feature that will resolve this.

Reply:
✅ Done in `9889eadf`, although I didn't fully understand. I was able to move the screen selection without those lines too. 

Regardless, it's now the same as in the `pixelate tool`.

Reply:
Thanks, what I meant was I could not move the inverted box the way other objects are moveable. This should fix it.